doc/en/developer/source/cite-test-guide/index.rst

```diff
@@ -8,226 +8,72 @@ A step by step guide to the GeoServer Compliance Interoperability Test Engine (C
-
-      * Ensure ``Recommendation Support`` is *checked*
-      * Ensure ``GML FeatureInfo`` is *checked*
-      * Ensure ``Fees and Access Constraints`` is *checked*
-      * For ``BoundingBox Constraints`` ensure ``Either`` is selected
-     
-   #. Click ``OK``
-
-   .. image:: tewms-1_1a.jpg
-
-   .. image:: tewms-1_1b.jpg
-
-Run WCS 1.1 tests
------------------
-
-#. Start GeoServer with the ``citewcs-1.1`` data directory.
-
-#. Change directory back to the cite tools and run the tests::
-
-     ant wcs-1.1
-     
-   With the following parameters:
-   
-   #. ``Capabilities URL``:
-
-         http://localhost:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.1.1
-   
-   Click ``Next``
-
-   .. image:: tewcs-1_1a.jpg
-
-#. Accept the default values and click ``Submit``
-
-   .. image:: tewcs-1_1b.jpg
-
-Run WCS 1.0 tests
------------------
-
-.. warning:: 
-
-   The WCS specification does not allow a cite compliant WCS 1.0 and
-   1.1 version to co-exist. To successfully run the WCS 1.0 cite tests
-   the ``wms1_1-<VERSION>.jar`` must be removed from the geoserver 
-   ``WEB-INF/lib`` directory.
+   .. note::
+      Valid values for the ETS parameter are
+        * wcs10
+        * wcs11
+        * wfs10
+        * wfs11
+        * wms11
+        * wms13
    
-#. Remove the ``wcs1_1-<VERSION>.jar`` from ``WEB-INF/lib`` directory.
-
-#. Start GeoServer with the ``citewcs-1.0`` data directory.
-
-#. Change directory back to the cite tools and run the tests::
-
-     ant wcs-1.0
-     
-   With the following parameters:
-
-   #. ``Capabilities URL``:
-        
-          http://localhost:8080/geoserver/wcs?service=wcs&request=getcapabilities&version=1.0.0
-
-   #. ``MIME Header Setup``: "image/tiff"
-
-   #. ``Update Sequence Values``:
-
-      * "2" for ``value that is lexically higher``
-      * "0" for ``value that is lexically lower``
-
-   #. ``Grid Resolutions``:
+   Choose which GeoServer war to test by setting the GEOSERVER_WEBAPP_SRC environment variable::
 
-      * "0.1" for ``RESX``
-      * "0.1" for ``RESY``
+    GEOSERVER_WEBAPP_SRC=https://build.geoserver.org/geoserver/master/geoserver-master-latest-war.zip
 
-   #. ``Options``:
+#. Build the testing suite::
   
-      * Ensure ``Verify that the server supports XML encoding`` is *checked*
-      * Ensure ``Verify that the server supports range set axis`` is *checked*
-
-   #. ``Schemas``:
-
-      * Ensure that ``original schemas`` is selected
-
-   #. Click ``OK``
-
-   .. image:: tewcs-1_0a.jpg
-
-   .. image:: tewcs-1_0b.jpg
-
-.. _commandline:
+    # FROM THE GEOSERVER CODE BASE ROOT FOLDER
+    pushd build/cite
+    docker_command="docker-compose -f docker-compose.yml -f ./${ETS}/docker-compose.override.yml "
+    ### Cleaning up previous runs
+    eval $docker_command  down --rmi all -v
+    eval $docker_command  rm -vfs
+    
+    ### Build docker images
+    eval $docker_command  build --build-arg "GEOSERVER_WEBAPP_SRC=${GEOSERVER_WEBAPP_SRC}" geoserver
```

Comment:
Is the `GEOSERVER_WEBAPP_SRC` environmental variable even used? Or is it only used by this eval command?

It may be worth while having a direct example of just what to type for the `wfs` service, or have a series of examples as with the developer manual [building](https://docs.geoserver.org/latest/en/developer/maven-guide/index.html#building) page for maven...

The above reads like a script (fine for jenkins) hard to follow by hand.